site/en/blog/soft-navigations-experiment/index.md

```diff
@@ -70,7 +70,17 @@ Site owners can choose to include the origin trial on their pages for all, or fo
 
 ## How can I measure Core Web Vitals per soft navigation?
 
-Some considerations need to be taken into account when attempting to measure Core Web Vitals for soft navigations.
+To includes soft navigations you need to include `includeSoftNavigationObservations: true` in your performance observer's `observe` call:
```

Comment:
Should we be explicit about this being the case starting from Chrome 112?

Reply:
I considered this, but since there's no harm of setting this now I'm fine with keeping it simple.